Test suite fails on Windows

* `babel-plugin-macros` version: 2.5.1
* `node` version:  both 8.x and 10.x
* `npm` version: 6.9.0

What you did:

`npm test`

What happened:

```
λ npm test

> babel-plugin-macros@0.0.0-semantically-released test C:\Projects\babel-plugin-macros
> kcd-scripts test
 PASS  src/__tests__/create-macros.js
 FAIL  src/__tests__/index.js
  ● Test suite failed to run

    Cannot find module 'babel-plugin-macros-test-fake/macro' from 'C:\Projects\babel-plugin-macros\src\__tests__'

      at Function.module.exports [as sync] (node_modules/resolve/lib/sync.js:69:15)
      at nodeResolvePath (node_modules/babel-plugin-macros/dist/index.js:49:18)
      at applyMacros (node_modules/babel-plugin-macros/dist/index.js:178:23)
      at VariableDeclaration.path.get.filter.forEach.child (node_modules/babel-plugin-macros/dist/index.js:126:30)
          at Array.forEach (<anonymous>)
      at VariableDeclaration (node_modules/babel-plugin-macros/dist/index.js:116:55)
      at NodePath._call (node_modules/@babel/traverse/lib/path/context.js:53:20)
      at NodePath.call (node_modules/@babel/traverse/lib/path/context.js:40:17)
      at NodePath.visit (node_modules/@babel/traverse/lib/path/context.js:88:12)
      at TraversalContext.visitQueue (node_modules/@babel/traverse/lib/context.js:118:16)
```

Problem description:

As mentioned in #111, the test suite fails on Windows.

I did a bit of troubleshooting and these are the findings:

* `mock-modules` are not present in the `node_modules` directory when `resolve.sync` for `'babel-plugin-macros-test-fake/macro'` fails 
* most likely `beforeAll` hook is not executed and the `mock-modules` are not copied properly

Now goes the funny part. I don't have a slightest idea what the execution model of the test code is and what kind of black magic happens there.

These are the things I don't get:

* Why does it fail inside the `babel-plugin-macros` code within `node_modules` and not the inside `src` directory?

* Why does it fail with the same error even if I add `throw new Error('In global scope!');` on top of `src/__tests__/index.js`?

* Why does it fail with the expected error if I add `throw new Error('In global scope!');` on top of `src/__tests__/index.js` and comment out the entire `pluginTester` invocation?

* Why wouldn't this happen in other environments?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,6 @@
+os:
+  - linux
+  - windows
 sudo: false
 language: node_js
 cache:

--- a/other/babel-config.js
+++ b/other/babel-config.js
@@ -1,14 +1,15 @@
 // we don't want to use babel-macros inside babel-macros...
 // and until kcd-scripts gets upgraded to babel-plugin-macros
 // we'll have both of those referenced here...
+const path = require('path')
 const getBabelConfig = require('kcd-scripts/babel')
 
 module.exports = (...args) => {
   const babelConfig = getBabelConfig(...args)
   babelConfig.plugins = babelConfig.plugins.filter(
     pluginPath =>
-      !pluginPath.includes('babel-macros/') &&
-      !pluginPath.includes('babel-plugin-macros/'),
+      !pluginPath.includes(path.normalize('babel-macros/')) &&
+      !pluginPath.includes(path.normalize('babel-plugin-macros/')),
   )
   const envPreset = babelConfig.presets.find(
     p => p[0] && p[0].includes('preset-env'),

